packages/ui-view/src/View/README.md

```diff
@@ -458,47 +458,212 @@ The focus outline adjusts to account for the shape of the View. For example, the
 to match the border radius of the corresponding View. For Views with irregular border radius (e.g., `borderRadius="small large none medium"`), the focus outline will appear with square edges. The color of the focus outline can be
 changed for different contexts via the `focusColor` property.
 
-```javascript
----
-type: example
----
-class FocusedExample extends React.Component {
-  constructor (props) {
-    super(props)
+- ```javascript
+  class FocusedExample extends React.Component {
+    constructor(props) {
+      super(props)
 
-    this.state = {
-      isFocused: true,
-      inset: false
+      this.state = {
+        isFocused: true,
+        inset: false
+      }
     }
-  }
 
-  updateFocused = (event) => {
-    this.setState({isFocused: event.target.checked})
-  }
+    updateFocused = (event) => {
+      this.setState({ isFocused: event.target.checked })
+    }
+
+    updateInset = (event) => {
+      this.setState({ inset: event.target.checked })
+    }
+
+    render() {
+      const { isFocused, focusPosition } = this.state
 
```

Comment:
Here these values are deconstructed but never used later. So i don't think this line is necessary.

Reply:
Removed